addons/building_markers/functions/fnc_handleObjectEdited.sqf

```diff
@@ -0,0 +1,25 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Handles editing of an object by Zeus.
+ *
+ * Arguments:
+ * 0: Curator (not used) <OBJECT>
+ * 1: Edited Object <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [_curator, _object] call zen_building_markers_fnc_handleObjectEdited
+ *
+ * Public: No
+ */
+
+params ["", "_object"];
+
+// Update building marker
+private _marker = _object getVariable [QGVAR(buildingMarker), ""];
```

Comment:
Maybe just `QGVAR(marker)` for variable name since it has the component prefix as well. In other places as well of course.

Reply:
https://github.com/zen-mod/ZEN/pull/704/files#diff-904878a4c3022a6347e5b85fe824f3641b2911718a3985159b1d6bdf0b3ad5e0R29